jpype/_jvmfinder.py

```diff
@@ -320,10 +320,10 @@ def _javahome_binary(self):
         """
         import platform
         import subprocess
-        from distutils.version import StrictVersion
+        from packaging.version import Version
 
-        current = StrictVersion(platform.mac_ver()[0][:4])
-        if current >= StrictVersion('10.6') and current < StrictVersion('10.9'):
+        current = Version(platform.mac_ver()[0][:4])
+        if current >= Version('10.6') and current < Version('10.9'):
```

Comment:
I guess this is part of your distutils handling. I suggest that this can be a separate PR, or alternatively, you should add ``packaging`` as an explicit dependency.

Reply:
Okay.  This was required to get Python 3.11 to work for me so I think it is same theme.  I will look at dependencies.

Reply:
Hmm.  I dont see where it goes in setup.py.  we never listed distutils as a requirement.  Can you suggest the required line.

Reply:
Sure. It is install_requires https://github.com/jpype-project/jpype/blob/master/setup.py#L81.

Reply:
I guess it was supposed to have distutils there before.   

https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/

I don't see the same format we have so I am guessing we want...

```  python
install_requires=['typing_extensions ; python_version< "3.8"',
        'packaging ; python_version< "3.10"'],
```

Reply:
distutils is a standard library, which has been deprecated for a while now and will be removed in Python 3.12.

Reply:
```
'packaging ; python_version< "3.10"'
```

Should just be

```
'packaging'
```